project_template/bin/setup

```diff
@@ -1,6 +0,0 @@
-#!/bin/sh
```

Comment:
Maybe leave `bin/setup` in place, because it is used quite often in project tooling (CI for example). Maybe just run `yarn install` from this file.

Reply:
Good point, will do.